### PR TITLE
Updates assertion link

### PR DESCRIPTION
**Pull request by iamnotaturtle**
Noticed that the current link 404s

### CODE REVIEW

File: src/guides/pet-shop.md [hunk 1]
```diff
@@ -309,7 +309,7 @@ Truffle is very flexible when it comes to smart contract testing, in that tests
 
 We start the contract off with 3 imports:
 
-* `Assert.sol`: Gives us various assertions to use in our tests. In testing, **an assertion checks for things like equality, inequality or emptiness to return a pass/fail** from our test. [Here's a full list of the assertions included with Truffle](https://github.com/trufflesuite/truffle/blob/master/packages/core/lib/testing/Assert.sol).
+* `Assert.sol`: Gives us various assertions to use in our tests. In testing, **an assertion checks for things like equality, inequality or emptiness to return a pass/fail** from our test. [Here's a full list of the assertions included with Truffle](https://github.com/trufflesuite/truffle/blob/develop/packages/resolver/solidity/Assert.sol).
```
Comment (cds-amal) [inline src/guides/pet-shop.md:312]: Links should reference the less volatile master branch.

File: src/tutorial.md [hunk 2]
```diff
@@ -316,7 +316,7 @@ Truffle is very flexible when it comes to smart contract testing, in that tests
 
 We start the contract off with 3 imports:
 
-* `Assert.sol`: Gives us various assertions to use in our tests. In testing, **an assertion checks for things like equality, inequality or emptiness to return a pass/fail** from our test. [Here's a full list of the assertions included with Truffle](https://github.com/trufflesuite/truffle/blob/master/packages/core/lib/testing/Assert.sol).
+* `Assert.sol`: Gives us various assertions to use in our tests. In testing, **an assertion checks for things like equality, inequality or emptiness to return a pass/fail** from our test. [Here's a full list of the assertions included with Truffle](https://github.com/trufflesuite/truffle/blob/develop/packages/resolver/solidity/Assert.sol).
```
Comment (cds-amal) [inline src/tutorial.md:319]: Same as above, please update the link to point to the master branch

**Review by cds-amal — CHANGES_REQUESTED** [review]
Thanks for this @iamnotaturtle ! We really appreciate the contribution. 🙇 . Just a small nit regarding the links.

File: src/guides/pet-shop.md (hunk 1 above)
Comment (iamnotaturtle) [reply src/guides/pet-shop.md]: Got it, updated!

File: src/tutorial.md (hunk 2 above)
Comment (iamnotaturtle) [reply src/tutorial.md]: updated 😉

**Review by cds-amal — APPROVED** [review]
Looks good! Thanks @iamnotaturtle !

**Review by eggplantzzz — APPROVED** [review]
I like it! Thanks @iamnotaturtle

**Pull request merged**